packages/strapi-helper-plugin/lib/internals/generators/container/index.js

```diff
@@ -38,13 +47,13 @@ module.exports = {
     name: 'wantSagas',
     default: true,
     message: 'Do you want sagas for asynchronous flows? (e.g. fetching data)',
-  }, {
-    type: 'confirm',
-    name: 'wantMessages',
-    default: true,
-    message: 'Do you want i18n messages (i.e. will this component use text)?',
   }],
   actions: (data) => {
+    const dataFormatted = data;
```

Comment:
Why do you need to declare a variable in this case? You could call your function parameter `dataFormatted` it will work as the same.